modules/dbnd-airflow/src/dbnd_airflow/backports/kubernetes_executor.py

```diff
@@ -0,0 +1,37 @@
+from distutils.version import LooseVersion
+
+import airflow
+
+
+if LooseVersion(airflow.version.version) > LooseVersion("1.10.10"):
+    from airflow.executors.kubernetes_executor import (
+        AirflowKubernetesScheduler,
+        KubernetesJobWatcher,
+        KubernetesExecutor,
+        KubeConfig,
+    )
+else:
+    from airflow.contrib.executors.kubernetes_executor import (
+        AirflowKubernetesScheduler,
+        KubernetesJobWatcher,
+        KubernetesExecutor,
+        KubeConfig,
+    )
+
```

Comment:
Possible option to avoid code duplication would be to
```py
module_path = "airflow.contrib.executors.kubernetes_executor" if AIRFLOW_10 else "airflow.executors.kubernetes_executor" 
executor_module = importlib.import_module(module_path)
AirflowKubernetesScheduler = getattr(executor_module, "AirflowKubernetesScheduler")
KubernetesJobWatcher = getattr(executor_module, "KubernetesJobWatcher")
KubernetesExecutor = getattr(executor_module, "KubernetesExecutor")
KubeConfig = getattr(executor_module, "KubeConfig")
```
but explicit over implicite... so I'm not sure